src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyRoleInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace FondOfSpryker\Zed\CompanyUserCompanyAssigner\Business\Model;
+
+use Generated\Shared\Transfer\CompanyUserTransfer;
+
+interface CompanyRoleInterface
```

Comment:
```suggestion
interface CompanyRoleManagerInterface
```

Reply:
@daniel-rose i've made the desired change